meson.build

```diff
@@ -27,7 +27,7 @@ project(
         # options :).
         'force_fallback_for=lz4,cjson',
     ],
-    meson_version: '>= 0.62.0'
+    meson_version: '>=1.0.1'
```

Comment:
I can't create a virtual environment with meson 1.0.1.  I get this error:
```
ERROR: No matching distribution found for meson==1.0.1
```

Reply:
I guess 1.0.1 isn't released yet.

Reply:
Yep. I just went ahead and posted this already. Not gonna be merged until next week.